pyatv/mrp/__init__.py

```diff
@@ -339,6 +339,13 @@ def artwork_id(self):
         return MrpPlaying(self.psm.playing)
 
 
+class PowerListener(PowerListener):
```

Comment:
No need for this class here as it's not used (only users outside of `pyatv` provides an implementation of this). The only reason for the `PowerListener` interface is so that it's easy to see which methods that should be implemented.

Reply:
I hope I've finally understood the Listeners idea.
It should be fixed now.